.circleci/config.yml

```diff
@@ -0,0 +1,147 @@
+version: 2.1
+
+aliases:
+  - &working_directory ~/merchant-center-application-kit
+
+  - &restore_yarn_cache
+    name: 'Restoring yarn cache'
+    keys:
+      - v1-yarn-cache-{{ checksum "yarn.lock" }}
+      - v1-yarn-cache
+
+  - &save_yarn_cache
+    name: 'Saving yarn cache'
+    key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+    paths:
+      - ~/.cache/yarn
+
+executors:
+  node_10:
+    docker:
+      - image: circleci/node:10.15
+    working_directory: *working_directory
+
+orbs:
+  cypress: cypress-io/cypress@1
+
+commands:
+  install_dependencies:
+    description: 'Installing dependencies'
+    steps:
+      - checkout
+      - restore_cache: *restore_yarn_cache
+      - run: yarn install --pure-lockfile
+      - save_cache: *save_yarn_cache
+  build_artifacts:
+    description: 'Building artefacts'
+    steps:
+      - run: yarn build
+  lint:
+    description: 'Running linters'
+    steps:
+      - run: yarn run jest --projects jest.eslint.config.js jest.stylelint.config.js --maxWorkers=3 --reporters jest-silent-reporter
+  unit_test:
+    description: 'Running tests'
+    steps:
+      - run: yarn run jest --projects jest.test.config.js --maxWorkers=3 --reporters jest-silent-reporter
+  vrt_test:
+    description: 'Running Visual Regression Tests'
+    steps:
+      - run: yarn run-if:percy
+  build_playground:
+    description: 'Building playground'
+    steps:
+      - run: yarn playground:build
+  build_starter:
+    description: 'Building starter'
+    steps:
+      - run: yarn template-starter:build
+  publish_to_canary:
+    steps:
+      - install_dependencies
+      - build_artifacts
+      - run:
+          name: Publishing to npm registry
+          command: ./scripts/release_canary.sh
+
+jobs:
+  lint:
+    executor: node_10
+    steps:
+      - install_dependencies
+      - lint
+  unit_test:
+    executor: node_10
+    steps:
+      - install_dependencies
+      - unit_test
+  vrt_test:
+    executor: node_10
+    steps:
+      - install_dependencies
+      - build_artifacts
+      - vrt_test
+  publish:
+    executor: node_10
+    steps:
+      - install_dependencies
+      - build_artifacts
+      - publish_to_canary
+
+workflows:
+  version: 2
+
+  build_and_test:
+    jobs:
+      - lint:
+          filters:
+            branches:
+              only:
+                - refactor/use-circleci
+      - unit_test:
+          requires:
+            - lint
+      - vrt_test:
+          requires:
+            - lint
+
+  e2e_test:
+    jobs:
+      - cypress/install:
+          name: cypress/install/e2e_playground
+          yarn: true
+          build: yarn playground:build
```

Comment:
Bleah. This internally uses yarn 1.10. We require > 1.13. It has to be used like this cause the orb interally persists to the workspace. Not sure what do to here.

Reply:
I think you can pass an option to yarn to ignore the check

Reply:
Thanks will try. Hoping it doesn't come from inside the orb itself.

Reply:
I don't think this is on "our level". It comes from within the orb [here](https://github.com/cypress-io/circleci-orb/blob/master/src/orb.yml#L83). I really like the orb though and would like to use it. Either we drop it (obviously) or we open a PR for them to accept `args: string` to the orb so we can pass `--ignore-engines`.